src/lib/services/tx/index.ts

```diff
@@ -219,4 +232,75 @@ export const useTxsCountByAddress = (
   );
 };
 
+export const useTxsByContractAddressLcd = (
+  address: BechAddr32,
+  offset: number,
+  limit: number
```

Comment:
pls change the seq to limit, offset

Also in below change as well.

Reply:
okay